karma-mocha-old.conf.js.tmpl

```diff
@@ -43,9 +43,6 @@ module.exports = function(config) {
       'jujugui/static/gui/src/test/test_controller_api.js',
       'jujugui/static/gui/src/test/test_env.js',
       'jujugui/static/gui/src/test/test_login.js',
-      'jujugui/static/gui/src/test/test_model_bundle.js',
-      'jujugui/static/gui/src/test/test_model_handlers.js',
-      'jujugui/static/gui/src/test/test_model.js',
```

Comment:
How do tests get loaded/excluded from the new suite? I'm curious since I just seem them being removed from old conf here but not added to the new one, so I'm assuming something automatic is happening?

Reply:
It uses a few different wildcards...

```
      'jujugui/static/gui/src/app/test-*.js': ['browserify'],
      'jujugui/static/gui/src/app/d3-components/test-*.js': ['browserify'],
      'jujugui/static/gui/src/app/init/**/test-*.js': ['browserify'],
```